src/libs/actions/Session/index.ts

```diff
@@ -92,7 +92,7 @@ Onyx.connect({
 let preferredLocale: ValueOf<typeof CONST.LOCALES> | null = null;
 Onyx.connect({
     key: ONYXKEYS.NVP_PREFERRED_LOCALE,
-    callback: (val) => (preferredLocale = val),
+    callback: (val) => (preferredLocale = val ?? null),
```

Comment:
We can change the initial value of `preferredLocale` to `undefined` and get rid of `null`

Reply:
Keeping this as `null` instead of `undefined` was intentional, because we use it (directly) in a lot of `API.write` calls.

Changing this to `undefined` would mean we need to add a lot of `?? null` on the other hand